tests/auth_tests/test_basic.py

```diff
@@ -4,7 +4,6 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AnonymousUser, User
 from django.contrib.auth.tests.custom_user import CustomUser
```

Comment:
I added a commit to move the non-documented auth models to the test package. It's a nice cleanup.